project_timeline/readme/USAGE.rst

```diff
@@ -0,0 +1,3 @@
+* Go to *Project > Search > Tasks* or *Project > Dashboard*.
+* Click on the icon with a clock.
```

Comment:
```suggestion
* Click on the icon with a timeline.
```
please change it because icon clock is view activity.

Reply:
> Functional Test, LGTM +1 But I don't know is it bug?
> 
>     * In v.12, I can mouse scroll up/down for zome in/out. But v.13 can't

I confirm that the zoom does not work even with the PR https://github.com/OCA/web/pull/1598

But I don't know if it's a bug either :)

Reply:
That's a vislibrary feature that may have changed, but has nothing to do with the modules themselves.